components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -4296,6 +4296,36 @@ def searchProcessing():
             search.close()
         return rv
 
+    def getThumbnailSet(self, image_ids, max_size=64):
```

Comment:
Needs docstring.
Also, we default to using size 96 for thumbs elsewhere so that would be preferable unless there's a reason to use 64?